cli/dcommunity.rb

```diff
@@ -1,9 +1,17 @@
-
+# Class decorating the Community Java Class
+# @see https://github.com/DSpace/DSpace/blob/dspace-5.3/dspace-api/src/main/java/org/dspace/content/Community.java
 class DCommunity
+
+  # Find a child collection by its name
+  # @param name [String]
+  # @return [Array<org.dspace.content.DSpaceObject>]
```

Comment:
I can't test locally, but since it's indexing `[0]`, I assume it returns a single DSO, not an array?